library_schemas/score/prerelease/HED_score_1.0.0.mediawiki

```diff
@@ -984,17 +984,14 @@ This project was supported by the by the National Institute of Mental Health of
             *** Clinical-EEG-temporal-relationship
                 **** Clinical-start-followed-EEG <nowiki>[Clinical start, followed by EEG start by X seconds.]</nowiki>
                     ***** <nowiki># {takesValue, valueClass=numericClass, unitClass=timeUnits}</nowiki>
-                    ***** <nowiki># {takesValue, valueClass=textClass}[free text]</nowiki>
                 **** EEG-start-followed-clinical <nowiki>[EEG start, followed by clinical start by X seconds.]</nowiki>
                     ***** <nowiki># {takesValue, valueClass=numericClass, unitClass=timeUnits}</nowiki>
-                    ***** <nowiki># {takesValue, valueClass=textClass}[Free text.t]</nowiki>
                 **** Simultaneous-start-clinical-EEG
                 **** Clinical-EEG-temporal-relationship-not-possible-to-determine <nowiki>[Not possible to determine.]</nowiki>
                     ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
 
             *** Episode-event-count <nowiki>[Number of stereotypical episodes during the recording.]</nowiki>
                 **** <nowiki># {takesValue, valueClass=numericClass} </nowiki>
-                **** <nowiki># {takesValue, valueClass=textClass}[Free text.t]</nowiki>
```

Comment:
Similar to Clinical-start-followed-EEG, should enable numeric entry AND free text.

How should this be enabled in the schema?

Reply:
See above.